Constants.js

```diff
@@ -48,10 +48,10 @@ const prettyRole = {
 
 const firebaseErrors = {
   "auth/email-already-in-use": "Email is already in use",
-  "auth/invalid-email": "Invalid email",
+  "auth/invalid-email": "Incorrect email / password",
```

Comment:
Is this being used anymore?

Reply:
This is tripped when an email that haven't been signed up is entered in signin.js. It's not explicitly mentioned but the error is returned by firebase

Reply:
But it's a good point because it's hard to know which is tripped and which is not. Only unit tests can tell